.github/workflows/lint.yml

```diff
@@ -1,42 +1,42 @@
-    steps:
-      - name: Checkout OpenSearch Dashboards
-        uses: actions/checkout@v2
-        with:
-          repository: opensearch-project/OpenSearch-Dashboards
-          ref: ${{ env.OPENSEARCH_DASHBOARDS_VERSION }}
-          path: OpenSearch-Dashboards
-      - name: Setup Node
-        uses: actions/setup-node@v3
-        with:
-          node-version-file: './OpenSearch-Dashboards/.nvmrc'
-          registry-url: 'https://registry.npmjs.org'
-      - name: Install Yarn
-        # Need to use bash to avoid having a windows/linux specific step
-        shell: bash
-        run: |
-          YARN_VERSION=$(node -p "require('./OpenSearch-Dashboards/package.json').engines.yarn")
-          echo "Installing yarn@$YARN_VERSION"
-          npm i -g yarn@$YARN_VERSION
-      - run: node -v
-      - run: yarn -v
-      - name: Checkout plugin
-        uses: actions/checkout@v2
-        with:
-          path: OpenSearch-Dashboards/plugins/dashboards-flow-framework
-      - name: Bootstrap the plugin
-        run: |
-          cd OpenSearch-Dashboards/plugins/dashboards-flow-framework
-          yarn osd bootstrap
-      - name: Run lint script
-        run: |
-          cd OpenSearch-Dashboards/plugins/dashboards-flow-framework
-          yarn lint:es common/* public/* server/*
+# name: Linting workflow
+# on: [push, pull_request]
+# env:
+#   OPENSEARCH_DASHBOARDS_VERSION: 'main'
+# jobs:
+#   run-lint:
+#     name: Run lint script
+#     runs-on: ubuntu-latest
+#     if: github.repository == 'opensearch-project/dashboards-flow-framework'
+#     steps:
+#       - name: Checkout OpenSearch Dashboards
+#         uses: actions/checkout@v2
+#         with:
+#           repository: opensearch-project/OpenSearch-Dashboards
+#           ref: ${{ env.OPENSEARCH_DASHBOARDS_VERSION }}
+#           path: OpenSearch-Dashboards
+#       - name: Setup Node
+#         uses: actions/setup-node@v3
+#         with:
+#           node-version-file: './OpenSearch-Dashboards/.nvmrc'
+#           registry-url: 'https://registry.npmjs.org'
+#       - name: Install Yarn
+#         # Need to use bash to avoid having a windows/linux specific step
+#         shell: bash
+#         run: |
+#           YARN_VERSION=$(node -p "require('./OpenSearch-Dashboards/package.json').engines.yarn")
+#           echo "Installing yarn@$YARN_VERSION"
+#           npm i -g yarn@$YARN_VERSION
+#       - run: node -v
+#       - run: yarn -v
+#       - name: Checkout plugin
+#         uses: actions/checkout@v2
+#         with:
+#           path: OpenSearch-Dashboards/plugins/dashboards-flow-framework
+#       - name: Bootstrap the plugin
+#         run: |
+#           cd OpenSearch-Dashboards/plugins/dashboards-flow-framework
+#           yarn osd bootstrap
+#       - name: Run lint script
+#         run: |
+#           cd OpenSearch-Dashboards/plugins/dashboards-flow-framework
+#           yarn lint:es common/* public/* server/*
```

Comment:
Instead of commenting the workflow, you can disable it from Github itself. [Ref](https://docs.github.com/en/actions/using-workflows/disabling-and-enabling-a-workflow#disabling-a-workflow)

Reply:
Oh awesome! I've just disabled from there. Closing this PR, thanks!